### ISSUE TITLE
Please remove all non-necessary notifications and popup

### ISSUE THREAD

**Issue by larsch** [dendronhq/dendron#3888]
Dendron is extremely noisy and nagful with notifications and popups that I do not want. If every extension sent this many notification, my screen would be filled with them and I am already weary of getting distracted by the ones from Dendron, and it hurts my productivity.

It's is a blatant misuse of the notification feature in VSCode.

## Please select if your request is either something new or an enhancement ##

- [ ] Enhancement of an existing Feature.
- [ ] Request of a new feature.
- [X] Request for removal of a "feature"

## Please select the area your request applies to. (Multiple selections are Possible. You can leave blank if you're not sure.) ##

- [X] Workspace - VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup - Dendron's Lookup Command
- [ ] Views - Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema - Dendron Schemas
- [ ] Pod - Data import from / export to Dendron
- [ ] Publish - External Site Publish
- [ ] Markdown - Markdown features, such as syntax support and features

## Is your feature request related to a problem? Please describe ##

I'm always frustrated when I receive unwanted notifications. They are distracting.

## Describe the solution you'd like ##

I would like to not receive notification unless my attention is required.

## Describe alternatives you've considered ##

Uninstalling Dendron.

**Comment by tlylt** [comment]
Just curious...what sort of notifications from Dendron is getting on your nerve? Could you provide some examples?

I personally feel that there are quite a number of popups from Dendron, but some of them seem to be useful/unavoidable. For example, the starting up `Dendron is active` feels annoying, but it tells me whether vscode is ready to execute Dendron commands so that I don't keep pressing some keyboard shortcuts and nothing happens. Other things like notifications on the progress for Dendron workspace sync feel like good UX too.

**Comment by larsch** [comment]
I can tolerate `Dendron is active`, as long as it is necessary for technical reasons. It's required for the usability of the extensions.

But generally, all the "have you heard about..." notifications about changed/new features are completely unwanted and pure noise for me. However, the drop that spilled the glass was logging into an older PC and getting "You don't seem to have used Dendron for a while" modal dialog.

Don't interrupt the user doing what he is trying to do. (I consider notifications an interruption, since they are distracting).

**Comment by v3ss0n** [comment]
I agree , many notifications are unnecessary , especially 90% of them in dendron.
